package.json

```diff
@@ -40,16 +40,23 @@
   },
   "devDependencies": {
     "enzyme": "^2.9.1",
-    "react-scripts": "1.0.7",
+    "eslint-plugin-babel": "^4.1.1",
+    "eslint-plugin-jsx": "0.0.2",
+    "eslint-plugin-react": "^7.0.1",
     "match-media": "^0.2.0",
+    "react-scripts": "1.0.7",
     "react-test-renderer": "^15.6.1"
   },
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
+    "lint-js": "eslint *.js \"src/**/*.js\"",
```

Comment:
This worked for me, don't know why it's not working here, but you can try specifying path to local eslint package `./node_modules/...`